snap/snapcraft.yaml

```diff
@@ -625,9 +625,12 @@ parts:
       source: https://github.com/sigstore/cosign.git
       go-importpath: github.com/sigstore/cosign/cmd/cosign
       source-commit: 33973d078170f586cf27f2cc464844b3f1fa1abb
+      build-environment:
+        - GOFLAGS:  "$GOFLAGS -modcacherw"
       override-build: |
         export GOROOT=/usr/local/go1.17
         export PATH="$GOROOT/bin:$PATH"
+        export GOFLAGS="${GOFLAGS} -mod=vendor -modcacherw"
```

Comment:
Since this is already specified in `build-environment`, I don't believe this is needed here.

Reply:
Weird, I thought I saw this same pattern in another package. Removing.

Reply:
you did, but you happened to pick the right (wrong?) example.  it was for another app that had some historical baggage around -mod=vendor.